rllib/core/testing/utils.py

```diff
@@ -62,6 +63,8 @@ def get_module_spec(framework: str, env: "gym.Env", is_multi_agent: bool = False
         observation_space=env.observation_space,
         action_space=env.action_space,
         model_config_dict={"fcnet_hiddens": [32]},
+        # TODO(Artur): Implement BCCatalog.
```

Comment:
Why do you need this? Catalog for RLModules is supposed to be an optional thing.

Reply:
It's a means for us to build our modules. It should not be enforce for user. That's why this unittest worked before the catalog were introduced.